Program.cs

```diff
@@ -1,21 +1,12 @@
 ﻿Console.Write ("Enter a number: ");
-string number = Console.ReadLine ();
-int num = int.Parse (number);
+int num = int.Parse (Console.ReadLine ());

```

Comment:
Fix this error.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/06bc5935-dba6-4603-b416-53eeeb91ce2d)